pom.xml

```diff
@@ -59,7 +59,6 @@
   <properties>
     <vertx.version>3.9.1</vertx.version>
     <rxjava.version>2.2.19</rxjava.version>
-    <slf4j-api.version>1.7.30</slf4j-api.version>
```

Comment:
I believe the documentation must be updated as it refers SL4JF.

Reply:
Removed it from the documentation, will add something on JBoss Logging (if applicable) in the next PR when I update the examples.